### PR TITLE
10 sort articles

### PR DESCRIPTION
**Pull request by CoupDeWhoop**

### CODE REVIEW

File: src/components/SortPanel.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { Link, useSearchParams } from 'react-router-dom'
+
+export default function SortPanel ({ order, toggleOrder }) {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const topic = searchParams.get("topic");
+
+  const path = `/articles/${topic ? `topic=${topic}` : ''}`
```
Comment (CoupDeWhoop) [inline src/components/SortPanel.jsx:10]: I've deleted this now. It was from an earlier attempt to deal with the search params

File: src/components/Nav.jsx [hunk 2]
```diff
@@ -19,27 +22,41 @@ const Nav = () => {
     })
   }, [])
 
+  const handleSortByMouseEnter = () => {
```
Comment (RichardNeat) [inline src/components/Nav.jsx:25]: We could maybe make this into one function to make your code a bit more DRY

File: utils/api.js [hunk 3]
```diff
@@ -10,8 +10,9 @@ export const getTopics = () => {
     })
 }
 
-export const getArticles = (topic) => {
-    return newsApi.get('/articles', { params: { topic }}).then(({data}) => {
+export const getArticles = (topic, sort_by, order) => {
+    console.log(topic, sort_by, order)
```
Comment (RichardNeat) [inline utils/api.js:14]: remove console log

**Review by RichardNeat — APPROVED** [review]
Nice one Dave 👏 this looks really nice and it's great that the url updates smoothly too :)

**Pull request merged**